examples/bearriver/src/FRP/BearRiver.hs

```diff
@@ -153,6 +155,21 @@ after q x = feedback q go
                   ct = if t' < 0 then constant (NoEvent, t') else go
               return ((e, t'), ct)
 
+occasionally :: MonadRandom m 
+            => Time -- ^ The time /q/ after which the event should be produced on average
```

Comment:
Please align?

Reply:
Done. So if splitting up types of a function into multiple lines, they all should align under the :: ?